doc/whats-new.rst

```diff
@@ -39,10 +39,14 @@ Enhancements
   attributes are retained in the resampled object. By
   `Jeremy McGibbon <https://github.com/mcgibbon>`_.
 
+- DataArray and Dataset methods :py:meth:`sel` and :py:meth:`loc` now
+  accept dictionaries or nested tuples for indexing on multi-index dimensions.
+  By `Benoit Bovy <https://github.com/benbovy>`_.
```

Comment:
Can you also please add a note about the changed behavior (we now drop levels, which is consistent with pandas) in the "Breaking changes" section above?

Also, add a reference here to the documentation section you added:

    :ref:`multi-level indexing`